docs/apm-input-settings.asciidoc

```diff
@@ -373,7 +373,7 @@ This token is not used for RUM endpoints.
 // tag::anonymous_enabled-setting[]
 |
 [id="input-{input-type}-anonymous_enabled-setting"]
-`anonymous_enabled`
+Anonymous Agent access
 
 | (bool) Enable or disable anonymous authentication.
```

Comment:
I believe that if this is set to `false` it will disable RUM as well, is that correct @axw ? If so, should we note that here or somewhere?

Reply:
Yeah, probably a good idea. Something like:

"RUM agents do not support authentication, so disabling anonymous access will effectively disable RUM agents."

Reply:
From `apm-server.yml`:
>    #anonymous:
      # By default anonymous auth is automatically enabled when either auth.api_key or
      # auth.secret_token is enabled, and RUM is enabled. Otherwise, anonymous auth is
      # disabled by default.

According to your comments, this will be disabled by default for the integration even when RUM and a token is configured?

Reply:
> this will be disabled by default for the integration even when RUM and a token is configured

 - A secret token is enabled
 - RUM is enabled

Therefore it will be automatically enabled, per the docs in apm-server.yml. Am I misunderstanding you?

Reply:
sorry, my bad - the default is `true`, so please ignore my comment.